lib/runner.js

```diff
@@ -185,6 +185,13 @@ var run = function (cmd, shOpts, opts, callback) {
     if (buffer) {
       process.stdout.write(stdout);
       process.stderr.write(stderr);
+      // Hack to help get `stdout|stderr` process streams flushed. We
+      // _shouldn't_ need to do this per:
+      // https://nodejs.org/api/process.html#process_process_stdout
+      // but here we are...
+      //
+      // TODO: https://github.com/FormidableLabs/builder/issues/120
+      console.log(""); // eslint-disable-line no-console
```

Comment:
Btw/, `process.stdout.write("")` didn't work. But by all accounts, the above is almost identical to it: https://github.com/nodejs/node/blob/master/lib/console.js#L39-L44

```js

Console.prototype.log = function(...args) {
  this._stdout.write(`${util.format.apply(null, args)}\n`);
};
```

So, we'll definitely categorize this as "very, very hacky" / "shouldn't help but it does"...